.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Run unit tests
+on: [pull_request]
+jobs:
+  unit_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ubuntu packages
+        run: sudo apt-get install libxml2-dev libxslt1-dev python-dev
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup Python cache
+        uses: actions/cache@v2
+        with:
+          path: venv
+          key: venv-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('**/requirements-dev.txt') }}
```

Comment:
Looking at the generated cache key, I don't think it is resolving the second variable. Could we try `${{ matrix.python-version }}` instead?

Reply:
Sure. Looks like we might need to do the same to the supplier frontend action as well

Reply:
This looks much better `Cache not found for input keys: venv-Linux-3.7-925d5c0df96c957f702d7e25a0e6eeeb79003666a14110b4f25509d648cc908c`